### PR TITLE
Fix issue where auto adjust was crashing for low eth accounts

### PR DESCRIPTION
**Pull request by jxom**
Fixes RNBW-3403

## What changed (plus any additional context for devs)

Seemed to be an issue where accounts with low eth balances would experience a crash upon pressing "Auto adjust" (when pressing "Max") on the swap screen. This occurred because the `eth balance - (transaction fee + padding)` evaluated to below zero which caused an exception when updating the swap amount.

### CODE REVIEW

**Review by osdnk — APPROVED** [review]

File: src/hooks/useSwapInputHandlers.ts [hunk 1]
```diff
@@ -43,27 +44,29 @@ export default function useSwapInputHandlers() {
...
-Would you like to auto adjust the balance to leave some ${inputCurrencyAddress.toUpperCase()}?`,
-          title: 'Are you sure?',
-        });
+            ],
+            message: `You are about to swap all the ${inputCurrencyAddress.toUpperCase()} available in your wallet. If you want to swap back to ${inputCurrencyAddress.toUpperCase()}, you may not be able to afford the fee.
+      
+  Would you like to auto adjust the balance to leave some ${inputCurrencyAddress.toUpperCase()}?`,
+            title: 'Are you sure?',
+          });
+        }
```
Comment (estebanmino) [inline src/hooks/useSwapInputHandlers.ts:69]: looking good, wondering if we could add an alert for the `else` case, seems like now the user will press and nothing will happen

**Review by estebanmino — APPROVED** [review]
🍰

**Comment by jinchung** [comment]
@jxom looks like esteban's comments about the text to languages hasn't been addressed yet?

**Comment by jxom** [comment]
@estebanmino – can i get one more check on this again? I added your comments

**Comment by BrodyHughes** [comment]
failed e2e. ill QA once it passes!

**Review by estebanmino — APPROVED** [review]

**Comment by jxom** [comment]
@BrodyHughes – should be good for QA now.

**Pull request merged**